x/feegrant/client/cli/tx.go

```diff
@@ -230,3 +230,11 @@ Example:
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+func getPeriodReset(duration int64) time.Time {
+	return time.Now().Add(getPeriod(duration))
```

Comment:
Question: don't we need to use `blockTime` here instead of `time.Now()`?

Reply:
We don't have access to `blockTime` in the clients.